exercises/nucleotide-count/description.md

```diff
@@ -1,11 +1,19 @@
-Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
+Most people inherit not only their socioeconomic class from their parents, but also DNA, a set of chemical instructions that influence how their bodies are constructed!
```

Comment:
```suggestion
Each of us inherits from our biological mother and father a set of chemical instructions known as DNA that influences how are bodies are constructed.
```
@cmcaine, just a suggestion.

Reply:
I like the suggestion!

Reply:
If I can't think of something I like better then I'll take this but replace mother and father with parents 👍

Reply:
```suggestion
Each of us inherits from our biological parents a set of chemical instructions known as DNA that influence how our bodies are constructed. All known life depends on DNA or something very similar!
```

I wanted it to still have a bit of a hook, and this'll do.

Reply:
> or something very similar!

What would that be?  I generally like your suggested change but this last part left me wondering what that similar thing could be.  Maybe it doesn't need to be stated.

Reply:
RNA can replicate on its own under certain conditions. Depends on what you define life as, really. Maybe best to leave it out to avoid distracting people!